lighthouse-core/test/audits/performance-budget-test.js

```diff
@@ -136,55 +129,6 @@ describe('Performance: Resource budgets audit', () => {
         expect(item.size).toBeGreaterThanOrEqual(items[index + 1].size);
       });
     });
-    describe('budget path', () => {
-      it('applies the last matching budget', async () => {
```

Comment:
`matchingBudget` isn't part of `performance-budget` anymore, but this test still seems like the best way to make sure `performance-budget` is following this behavior (and calling `matchingBudget` internally)